File: 3925-concatenate-array-with-reverse/3925-concatenate-array-with-reverse.py
class Solution(object):
    def concatWithReverse(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        size = len(nums)
        res = [0] * (2 * size)

        for i in range(size):
            res[i] = res[2 * size - i - 1] = nums[i]
        
        return res

File: 3925-concatenate-array-with-reverse/3925-concatenate-array-with-reverse.cpp
class Solution {
public:
    vector<int> concatWithReverse(vector<int>& nums) {
        int size = nums.size();
        vector<int> res(2 * size);

        for(int i = 0 ; i < size ; i++)
            res[i] = res[2 * size - i - 1] = nums[i];

        return res;
    }
};